### DISCUSSION TITLE
System.import is not a function

### DISCUSSION THREAD

**Discussion by knightswatch3** [webpack/webpack#12980] [Q&A] [answered]
In middle of my migration from webpack 3 to 5 and I experience this following issue in my react project.

`Uncaught TypeError: System.import is not a function

**The above error occurred in the <Router> component:
    in Router (created by App)
    in Provider (created by App)
    in App
    in AppContainer**

React will try to recreate this component tree from scratch using the error boundary you provided, AppContainer.`

Any thoughts on how to fix it for my react code-base ?

**Comment by sokra** [comment, marked as answer]
Omit the `System.` in the code. That's outdated

**Reply by knightswatch3** [reply]
So just the import would do the job now ?

**Reply by knightswatch3** [reply]
That worked. Thanks.
